test/functionalTest/cases/0876_attribute_dates/attrs_dates_overridden_by_user_subs.test

```diff
@@ -33,9 +33,9 @@ accumulatorStart --pretty-print
 #
```

Comment:
No actual changes in this file? Only cleanup?

Reply:
... it doesn't seem so. I see:

```
Content-Length: REGEX((400|262))
```

A bit weird... why this PR makes that change needed?

Reply:
This is**truly** weird. We need to take a closer look at it.
The exact same payload (except whitespace, I assume) is sent to the subscriber with two very different Content-Lengths ...

Reply:
It seems to depend on timing.
But, if I remember correctly, look at the two notifications (one with Content-Length of 400, the other with 262) and you'll see that they are identical, except for the Content-Length ... :-O

Reply:
In my next PR I have another example of "exact same"  notification with two different Content-Lengths ...
Something is rotten in the kingdom of Denmark ...

We need to look into this. Not in this PR though, creating an issue

Reply:
https://github.com/telefonicaid/fiware-orion/issues/3258